### ISSUE TITLE
No error for prealloc even if it reports many

### ISSUE THREAD

**Issue by ferhatelmas** [alecthomas/gometalinter#451]
Running [prealloc](https://github.com/alexkohler/prealloc) directly reports many issues. However, if we run it through gometalinter, no issues are reported.

> prealloc ./...

```
src/github.com/agflow/web/go-lib/pgu/simple_types.go:222 Consider preallocating result
src/github.com/agflow/web/go-lib/util/agslice/slice.go:58 Consider preallocating r
src/github.com/agflow/web/go-lib/util/agslice/slice.go:67 Consider preallocating r
src/github.com/agflow/web/go-lib/util/agstring/string.go:340 Consider preallocating res
src/github.com/agflow/web/go-lib/workerutil/upload.go:148 Consider preallocating results
src/github.com/agflow/web/go-lib/workerutil/workerio/email.go:37 Consider preallocating reports
src/github.com/agflow/web/go-lib/workerutil/workerio/email.go:45 Consider preallocating reports
src/github.com/agflow/web/server/tender.go:113 Consider preallocating names
src/github.com/agflow/web/server/user.go:792 Consider preallocating fullnames
src/github.com/agflow/web/server/user.go:791 Consider preallocating profiles
....
```
> gometalinter --config gometalinter.json --enable=prealloc ./...

No error is reported.

Contents of config file (uninteresting parts are excluded):

```json
{
    "Vendor": true,
    "Deadline": "10m",
    "Linters": {
      "prealloc": {
        "Command": "prealloc -set-exit-status",
        "Pattern": "PATH:LINE:COL:MESSAGE"
      }
    },
    "LineLength": 96,
    "EnableGC": true,
    "Test": true,
    "Enable": []
}
```
I use `prealloc` as an example but I have seen similar issues in `errcheck`, `lll` and `structcheck` since `prealloc` is an external linter, thought it would be the best example.

**Comment by dnephin** [comment]
Can you include the output with `--debug` ? It's possible that you might need to set a partition strategy if the default isn't working.

I also think the `PATTERN` is wrong. There is no column in that output, so it should be `PATH:LINE:MESSAGE`. If that's the issue there should have been an error somewhere that apparently isn't being reported.

**Comment by ferhatelmas** [comment]
@dnephin With your suggestion, I was able to see the errors.

My update config is for `Linters` part:

```json
"prealloc": {
  "Command": "prealloc -set_exit_status",
  "Pattern": "^(?P<path>.*?\\.go):(?P<line>\\d+)\\s*(?P<message>.*)$"
}
```
